core/src/main/java/org/apache/iceberg/TableProperties.java

```diff
@@ -135,21 +135,31 @@ private TableProperties() {
   public static final String OBJECT_STORE_ENABLED = "write.object-storage.enabled";
   public static final boolean OBJECT_STORE_ENABLED_DEFAULT = false;
 
+  /**
+   * @deprecated will be removed in 0.14.0, use {@link #WRITE_DATA_LOCATION} instead
```

Comment:
I doubt this will ever be removed. It isn't worth breaking on older tables.

Reply:
Make sense, remove the comments.